float32 target_location_x
float32 target_location_y
float32 target_speed
uint8 task_direction
uint8 task_route_id

uint8 default_idle = 0
uint8 positive_direction = 1
uint8 opposite_direction = 2
